src/api/app/helpers/webui/request_helper.rb

```diff
@@ -129,4 +129,13 @@ def reviewer(review)
     return "#{review[:by_project]} / #{review[:by_package]}" if review[:by_package]
     review[:by_user] || review[:by_group] || review[:by_project]
   end
+
+  def diff_data(action_type, sourcediff)
+    diff = (action_type == :delete ? sourcediff['old'] : sourcediff['new'])
+    diff.slice('project', 'package', 'rev').with_indifferent_access
```

Comment:
using `with_indifferent_access` when you only want to use it with symbols doesn't seems like a good idea...

Reply:
Well, the sliced data is using strings as keys. And the render call expects symbols

Reply:
all render calls are using symbols... so.. you should use symbols... otherwise Ruby need to convert your symbols to strings.... :sob:

Reply:
`{ project: diff[:project], package: diff[:package], rev: diff[:rev] }`

Reply:
I doubt this has any relevant performance impact... but I don't want to have another bikeshed discussion. I changed it.